velox/exec/PlanNodeStats.cpp

```diff
@@ -55,8 +55,8 @@ std::string PlanNodeStats::toString(bool includeInputStats) const {
         << "), ";
   }
   out << "Output: " << outputRows << " rows (" << outputBytes << " bytes)"
-      << ", Cpu time: " << cpuWallTiming.cpuNanos << "ns"
-      << ", Blocked wall time: " << blockedWallNanos << "ns"
+      << ", Cpu time: " << cpuWallTiming.cpuNanos * 1.0 / 1'000 << "ms"
```

Comment:
We need to add proper logic to format these numbers by checking how large the number is and decidig whether to report it using ns, ms, s or min.

Reply:
Makes sense. I will look into this logic in a separate PR. Will revert this for now.